### DISCUSSION TITLE
Cloudlog & WSJT-X

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#2015] [Q&A]
I am still trying to get the WSJT-X  IMPROVED  PLUS to work with Cloudlog. Has anyone been successful and, if so, what is the API URL and API Key? 

Thanks & 73, Scott W1AL

**Comment by phl0** [comment]
For sure. It is running on multiple instances here. The URL is the base URL to your Cloudlog instance. The API key has to be generated in Cloudlog and the Station ID is the ID of the Logbook you want to send your logs to.

![Screenshot from 2023-02-24 15-52-44](https://user-images.githubusercontent.com/7112907/221209564-2eb1d133-504c-4acc-8e40-c0f0a5c29bc0.png)

**Reply by soakland5** [reply]
Still invalid API key.......I tried https://w1al.cloudlog.co.uk with  a copies and pasted API key. Station ID correct also  1

**Reply by phl0** [reply]
Do you have access to the logs?

**Reply by phl0** [reply]
Ah wait. Which operating system are you using? Is it a Windows machine?

**Reply by soakland5** [reply]
Yes it is. Windows 11

**Comment by MikleBas** [comment]
Yes, if you use an intermediary program like GridTracker
It works for me without problems, I recommend

**Reply by soakland5** [reply]
Yes I got that to work as well but I have low memory situation and trying to eliminate Gridtracker.... :)

**Comment by MikleBas** [comment]
I had a similar problem, the CloudCat program does not understand the HTTPS protocol
specify the non-HTTPS (HTTP) address in the server and token address
if your server forcibly redirects from http to https, then disable this function on the server - it should help

![cloudlog](https://user-images.githubusercontent.com/99086477/221222933-ef91dfd7-8c2b-44b2-9aed-65a187c1b82b.png)

**Reply by phl0** [reply]
This is probably a lack of SSL libs. At least for WSJT-X improved it works if you install Win OpenSSL. See https://github.com/magicbug/Cloudlog/wiki/WSJT-X-Integration#wsjt-x-improved-plus-versions-by-dg2ycb

**Reply by soakland5** [reply]
Does this work only for the server version of Cloudlog, or the web-based version?

**Reply by phl0** [reply]
I don't understand the question. Cloudlog is always web-based?! Do you mean self-hosted vs. cloudlog.co.uk?
If so, should not make a difference.

**Reply by soakland5** [reply]
I meant cloudlog.co.uk

**Reply by phl0** [reply]
It should work wherever Cloudlog is hosted. Did you install the Win OpenSSL package as I recommended earlier?

**Reply by soakland5** [reply]
Yes I just did. No difference. Even after running it. Is there a particular method / order of running it? Should I leave the "S" off of http  ??

**Reply by MikleBas** [reply]
No, leave HTTP address
without HTTPS
